segmentation/utils/spatial_analysis.py

```diff
@@ -53,202 +52,203 @@ def calculate_channel_spatial_enrichment(dist_matrices, marker_thresholds, all_d
                              "perimeter", "fov"]
 
     # Error Checking
-    if not np.isin(excluded_colnames, all_data.columns).all():
+    if not np.isin(excluded_colnames, all_fov_data.columns).all():
         raise ValueError("Column names were not found in Expression Matrix")
 
-    if not np.isin(fovs, all_data[fov_col]).all():
+    if not np.isin(included_fovs, all_fov_data[fov_col]).all():
         raise ValueError("Points were not found in Expression Matrix")
 
-    # Subsets the expression matrix to only have marker columns
-    data_markers = all_data.drop(excluded_colnames, axis=1)
-    # List of all markers
-    marker_titles = data_markers.columns
-    # Length of marker list
+    # Subsets the expression matrix to only have channel marker columns
+    all_channel_data = all_fov_data.drop(excluded_colnames, axis=1)
+    # List of all channel markers
+    marker_titles = all_channel_data.columns
+    # Length of channel marker list
     marker_num = len(marker_titles)
 
-    # Check to see if order of marker thresholds is same as in expression matrix
+    # Check to see if order of channel marker thresholds is same as in expression matrix
     if not (list(marker_thresholds.iloc[:, 0]) == marker_titles).any():
         raise ValueError("Threshold Markers do not match markers in Expression Matrix")
 
     # Create stats Xarray with the dimensions (points, stats variables, number of markers, number of markers)
     stats_raw_data = np.zeros((num_fovs, 7, marker_num, marker_num))
-    coords = [fovs, ["z", "muhat", "sigmahat", "p_pos", "p_neg", "h", "p_adj"], marker_titles,
+    coords = [included_fovs, ["z", "muhat", "sigmahat", "p_pos", "p_neg", "h", "p_adj"], marker_titles,
               marker_titles]
     dims = ["points", "stats", "marker1", "marker2"]
     stats = xr.DataArray(stats_raw_data, coords=coords, dims=dims)
 
     # Subsetting threshold matrix to only include column with threshold values
     thresh_vec = marker_thresholds.iloc[:, 1]
 
-    for i in range(0, len(fovs)):
-        # Subsetting expression matrix to only include patients with correct label
-        patient_ids = all_data[fov_col] == fovs[i]
-        fov_data = all_data[patient_ids]
-        # Patients with correct label, and only columns of markers
-        fov_channel_data = data_markers[patient_ids]
+    for i in range(0, len(included_fovs)):
+        # Subsetting expression matrix to only include patients with correct fov label
+        current_fov_idx = all_fov_data[fov_col] == included_fovs[i]
+        current_fov_data = all_fov_data[current_fov_idx]
```

Comment:
I think current_fov_data makes sense. But if we keep the master file as all_data, then this becomes even easier to read, since it will go all_data -> current_fov_data, emphasizing the difference between them